gammapy/detect/tests/test_cwt.py

```diff
@@ -5,7 +5,7 @@
 from ...utils.testing import requires_dependency, requires_data
 from ...detect import CWT, CWTKernels, CWTData
 from ...datasets import load_poisson_stats_image
```

Comment:
@hsiejkowski - the test still uses `load_poisson_stats_image`, which uses SkyImage, so we'd like to delete it.

Could you please update the tests to use Map to load the image file instead?

This should work:
```
def load_images():
    filename = '$GAMMAPY_EXTRA/test_datasets/unbundled/poisson_stats_image/counts.fits.gz'
    counts = Map.read(filename)
    background = counts.copy(data=np.ones(counts.data.shape, dtype=float))
    return dict(counts=counts, background=background)
```
I see there's several places below where the images are used. Probably we should use pytest.fixtures here, but possibly that's a bit complicated.

Maybe for now you just avoid the use of load_poisson_stats_image / SkyImage in the test somehow?

If you don't want to do this now, I can also do this in a follow-up commit tonight.

Apart from this and the geom copy mentioned in my last comment, I think this PR is ready to go.

Reply:
Done in e20da158291756bb076d94d7b2e6400b76db7142